dll/win32/shell32/lang/en-US.rc

```diff
@@ -355,24 +355,23 @@ FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
 END
 
-IDD_RUN_AS DIALOGEX 0, 0, 240, 190
-STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
+IDD_RUN_AS DIALOGEX 0, 0, 240, 182
+STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
 CAPTION "Run As"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
-    LTEXT "Which user account do you want to use to run this program?", -1, 10, 20, 220, 20
-    CHECKBOX "Current User %s", 14000, 10, 45, 150, 10
-    LTEXT "Protect my computer and data from unauthorized program activity", -1, 25, 57, 200, 10, WS_DISABLED
-    CHECKBOX "This option can prevent computer viruses from harming your computer or personal data, but selecting it might cause the program to function improperly.", 14001, 25, 68, 200, 30, WS_DISABLED | BS_MULTILINE
-    CHECKBOX "The following user:", 14002, 10, 100, 90, 10
-    LTEXT "User name:", -1, 20, 118, 54, 10
-    COMBOBOX 14003, 75, 115, 100, 15, CBS_DROPDOWNLIST | WS_VSCROLL | WS_VISIBLE | WS_TABSTOP
-    PUSHBUTTON "...", 14004, 180, 115, 30, 14, WS_TABSTOP
-    LTEXT "Password:", -1, 20, 143, 53, 10
-    EDITTEXT 14005, 74, 140, 100, 14, ES_LEFT | WS_BORDER | WS_GROUP
-    PUSHBUTTON "...", 14006, 180, 140, 30, 14, WS_TABSTOP
-    PUSHBUTTON "OK", 14007, 57, 170, 60, 14, WS_TABSTOP
-    PUSHBUTTON "Cancel", 14008, 122, 170, 60, 14, WS_TABSTOP
+    ICON IDI_SHELL_SECURITY, -1, 7, 7, 21, 20
+    LTEXT "Which user account do you want to use to run this program?", -1, 32, 12, 200, 16
+    AUTORADIOBUTTON "&Current user %s", IDC_RUNAS_THIS, 7, 35, 226, 10, WS_GROUP
+    AUTOCHECKBOX "&Protect my computer and data from unauthorized program activity\n\nThis option can prevent computer viruses from harming your computer or personal data, but selecting it might cause the program to function improperly.", IDC_RUNAS_SAFER, 18, 50, 210, 62, WS_DISABLED | WS_TABSTOP | BS_MULTILINE | BS_TOP
+    AUTORADIOBUTTON "The &following user:", IDC_RUNAS_OTHER, 7, 110, 226, 10
+    LTEXT "&User name:", -1, 18, 124, 54, 10
+    COMBOBOX IDC_RUNAS_NAME, 75, 122, 134, 40, CBS_DROPDOWN | CBS_AUTOHSCROLL | WS_VSCROLL | WS_VISIBLE | WS_TABSTOP
```

Comment:
or `IDC_RUNAS_USERNAME`
See for example, https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/msgina/lang/en-US.rc;h=68da6e2a0ff59f683b804884c075aa41d712901b;hb=HEAD#l31